mars/deploy/oscar/pool.py

```diff
@@ -51,9 +51,26 @@ def _parse_file_logging_config(
     formatter: Optional[str] = None,
     from_cmd: bool = False,
 ) -> configparser.RawConfigParser:
-    root_level, root_fmt = _get_root_logger_level_and_format()
-    level = level or root_level
-    formatter = formatter or root_fmt
+    """
+    If env is ipython (from_cmd=False), the log level and format on the web follow our default configuration file,
+    and the level and format on the console use the user's configuration (logging.basicConfig) or keep the default.
+
+    If env is cmd (from_cmd=True, e.g. user invokes `python -m mars.worker`),
+    the log level and format on the web and console follow user's config (--log-level and --log-format)
+    or our default configuration file.
+
+    Parameters
```

Comment:
Better remove the `Parameters` and `Returns` sections.

Reply:
Fix